src/Microphysics2M.jl

```diff
@@ -488,8 +488,11 @@ function radar_reflectivity(
     N_liq *= FT(1e-9)
     N_rai *= FT(1e-9)
 
-    xc = (N_liq == 0) ? FT(0) : ((q_liq * ρ_air) / N_liq)
-    xr = (N_rai == 0) ? FT(0) : ((q_rai * ρ_air) / N_rai)
+    q_liq = (q_liq < maximum(q_liq) * 1e-8) ? FT(0) : q_liq
```

Comment:
`q_liq` is just a number here, so I think that `maximum(q_liq)` is not really doing anything here

Reply:
Maybe it would be good to make the cutoffs for N and q the arguments of the Z and r_eff? Those hard-coded limits sometimes actually matter. So it would be good to be able to test that the thresholds we are choosing don't affect the output

Reply:
I agree that it's better to put them as arguments. It allows us to always check that the threshold do not alter the results; and considering that these two products are very sensitive to small modifications of their arguments I think it makes a lot more sense like this.

Reply:
I made the corrections and I also just squashed and rebased. Check that I did it correctly (but I think so), and lmk if the corrections look good to you! :)